src/Factory.php

```diff
@@ -25,17 +25,18 @@
  */
 abstract class Factory
 {
-    /** @var Attributes[] */
+    /** @phpstan-var Attributes[] */
```

Comment:
everythime we use phpstan pseudo types, we should use `@phpstan-` annotations because Psalm seems to not understand it

and moreover, see this comment from stof [here](https://github.com/symfony/symfony/pull/50687#discussion_r1281529038), I think it is relevant